ew/static/smelting.py

```diff
@@ -336,7 +365,8 @@
         ],
         ingredients={
             ewcfg.item_id_monsterbones: 100,
-            ewcfg.item_id_faggot: 1
+            'scalp':100,
```

Comment:
as a rule, we wouldn't use souls as crafting materials with the possible exception of some in-game events. please change this back

Reply:
Its allready a crafting material on reanimated corpses, will change for now

Reply:
true, but you can pull souls back out of those. the idea is that there shouldn't be an incentive to permanently destroy them